apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -1381,7 +1395,7 @@ static void utc_audio_pcm_mmap_write_p(void)
 		num_read = read(fd, buffer, size);
 		if (num_read > 0) {
 			ret = pcm_mmap_write(g_pcm, buffer, pcm_bytes_to_frames(g_pcm, num_read));
-			TC_ASSERT_GEQ_CLEANUP("pcm_mmap_write", ret, 0, clean_all_data(fd, buffer));
+			TC_ASSERT_CLEANUP("pcm_mmap_write", ret >= 0 || ret == -EPIPE, clean_all_data(fd, buffer));
```

Comment:
pcm_mmap_write also byte based operation

Reply:
Also below should be added.
if (ret == -EPIPE) {
    pcm_prepare(); //add this later
    continue; //for now
}

Reply:
I fixed the code for your first comment.
Regarding the second comment, the loop will anyway continue as of now. I will add the pcm_prepare check later.